client/my-sites/email/email-providers-in-depth-comparison/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { translate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { InDepthComparison, ProviderComparison } from './in-depth-comparison';
```

Comment:
Note we usually try to use absolute paths (e.g. this makes it easier to grok them).